notebooks/README.md

```diff
@@ -10,6 +10,7 @@ For detailed specifics of any concept mentioned look at the [Forecast developer
   - [query and evaluate one item at a time](https://github.com/aws-samples/amazon-forecast-samples/blob/master/notebooks/basic/Tutorial/3.Evaluating_Your_Predictor.ipynb)
   
 - Advanced folder contains notebooks to show API calls for more complex tasks:
+  - [Time Series Clustering Preprocessing](https://github.com/aws-samples/amazon-forecast-samples/blob/master/notebooks/advanced/Clustering_Preprocessing/02.%20Time%20Series%20Clustering%20Using%20DTW%20KMeans.ipynb)
```

Comment:
May be better to link to README file, because it contains the overview information and links to 2 notebooks.

Reply:
Nice suggestion, updated